corehq/apps/importer/templates/importer/excel_fields.html

```diff
@@ -12,7 +12,9 @@
 
                 if (this.checked) {
                     var excelField = $(this).closest('tr').find(".excel_field");
-                    customField.val(excelField.val().replace(/[^\w\s]/gi, ''));
+                    var escapedValue = excelField.val().replace(/[^\w\s]/gi, '');
+                    escapedValue = escapedValue.replace(/[\s]/gi, '_');
```

Comment:
before we weren't replacing ` ` with underscores when converting an excel column header and that's how we ended up with case properties with spaces in them. this ensures that the conversion converts the whitespace